src/integrationTest/java/uk/gov/hmcts/darts/task/service/AdminUpdateAutomatedTaskServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.darts.task.service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.darts.tasks.model.AutomatedTaskPatch;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SuppressWarnings("PMD.ExcessiveImports")
+class AdminUpdateAutomatedTaskServiceTest extends IntegrationBase {
+
+    @Autowired
+    private AdminAutomatedTaskService adminAutomatedTaskService;
+
+    @Test
+    void updatesIsEnabledFieldFalse() {
```

Comment:
What is the point of having 2 integration tests here? The assertions look to be testing the exact same thing except for the 'getTaskEnabled' check. It's not clear to me what we are trying to achieve here by having these in separate tests. Do we need both?

Reply:
When theres just 2 possible results I just think may as well cover them both given its quick to write and run.  Can just have 1 but I think the test name should change to be more generic.